OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1456,7 +1639,10 @@ public static DreamValue NativeProc_shutdown(DreamObject instance, DreamObject u
         [DreamProcParameter("format", Type = DreamValueType.String)]
         public static DreamValue NativeProc_time2text(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
             int timestamp = arguments.GetArgument(0, "timestamp").GetValueAsInteger();
-            string format = arguments.GetArgument(1, "format").GetValueAsString();
+            if (!arguments.GetArgument(1, "format").TryGetValueAsString(out var format))
+            {
+                return new DreamValue("");
```

Comment:
A null format argument returns a string in the format of `Thu Dec 30 15:13:20 1999`